src/josepy/util.py

```diff
@@ -158,6 +163,32 @@ def public_key(self):
         return self.__class__(key)
 
 
+class ComparableOKPKey(ComparableKey):
+    """Wrapper for ``cryptography`` OKP keys.
+
+    Wraps around:
+    - :class:`~cryptography.hazmat.primitives.asymmetric.ed25519.Ed25519PublicKey`
+    - :class:`~cryptography.hazmat.primitives.asymmetric.ed25519.Ed25519PrivateKey`
+    - :class:`~cryptography.hazmat.primitives.asymmetric.ed448.Ed448PublicKey`
+    - :class:`~cryptography.hazmat.primitives.asymmetric.ed448.Ed448PrivateKey`
+    - :class:`~cryptography.hazmat.primitives.asymmetric.x25519.X25519PublicKey`
+    - :class:`~cryptography.hazmat.primitives.asymmetric.x25519.X25519PrivateKey`
+    - :class:`~cryptography.hazmat.primitives.asymmetric.x448.X448PublicKey`
+    - :class:`~cryptography.hazmat.primitives.asymmetric.x448.X448PrivateKey`
+    """
+
+    def __hash__(self):
+        return hash((self.__class__, self._wrapped.curve.name, self._wrapped.x))
```

Comment:
Not sure what the `__hash__` should be defined as. The curve name cannot be obtained like this.